src/OpenDiffix.Core/NodeUtils.fs

```diff
@@ -104,7 +104,7 @@ let inline collect func node =
 let inline unwrap node =
   callUnwrap (Unchecked.defaultof<NodeFunctions>, node)
 
-/// Collects all aggregator expressions from a node.
+/// Collects all aggregators from expressions being immediate children of `node`.
```

Comment:
"immediate children of `node`" is not quite accurate. This recurses inside expressions, so if you have the following tree: `Func.Add (Agg.Count, Agg.CountDistinct)` it will collect both of the aggs.

Reply:
hm, but then `Func.Add` is an expression being the immediate child. If we have `Func.Add (Agg.Count, Func.Add (Agg.CountDistinct, 3))`, then the top most `Func.Add` is the expression being the immediate child. All aggregators from it will be collected/visited.

To me the current wording implies a "too deep" collection. But I'm new to this, so I'm OK with dropping, if the change sounds off.

Reply:
If the query has
```
SELECT:
  Func.Add (
    Agg.Count,
    1
  )
```

then we collect the aggregator from `SELECT::Add::Count`, which is not exactly the immediate child expression.

Reply:
ah, ok, I was parsing this like `Count` _is_ the aggregator, and it is _from_ `SELECT::Add` being immediate child expr. Hm... any counter suggestions to rephrase the original to not sound like traversing everything `from a node`? :thinking:

Reply:
Not sure for alternatives, but on callsites it will be pretty obvious what is going on.

Reply:
To me, it is obvious (or at least clear) now, but wasn't initially. But let's undo the change :+1: